src/main/java/com/app/katchup/MeetingResponse/MeetingInboxResponseController.java

```diff
@@ -33,4 +40,19 @@
         logger.info(String.format("Returning %s meeting invites for user:%s", invites.size(), userName));
         return new ResponseEntity<>(invites, HttpStatus.OK);
     }
+
+    @PutMapping("/meetings/{meetingId}/response")
+    public ResponseEntity<Decision> putInboxForUserDecision(@PathVariable String meetingId,
```

Comment:
```suggestion
    public ResponseEntity<Decision> putUserDecisionForMeetingInviteResponse(@PathVariable String meetingId,
``` 
Please rename it to putUserDecisionForMeetingInviteResponse